src/adagrams.js

```diff
@@ -1,15 +1,107 @@
+const letterPool = {
+  'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3,
+  'H': 2, 'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6,
+  'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4,
+  'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+};  
+  
+const letterValue = new Map([
+  [['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'], 1],
+  [['D', 'G'], 2],
+  [['B', 'C', 'M', 'P'], 3],
+  [['F', 'H', 'V', 'W', 'Y'], 4],
+  [['K'], 5],
+  [['J', 'X'], 8],
+  [['Q', 'Z'], 10]
+]);
+
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+
+let lettersList = [];
+for (const [letter, count] of Object.entries(letterPool)) {
+    for (let i = 0; i < count; i++) {
+        lettersList.push(letter);
+    }
+}
+
+let hand = [];
+let temporaryLettersList = [...lettersList];
+
+for (let i = 0; i < 10; i++) {
+    const randomIndex = Math.floor(Math.random() * temporaryLettersList.length);
+    const randomLetter = temporaryLettersList.splice(randomIndex, 1)[0];
```

Comment:
These are letter counts will the same size but what if they weren't? What if the input size varied? What would be the time complexity of your code? What CS fundamentals data structure did we learn about that is handy for keeping track of the number of occurrences?